kore/src/Kore/Rewrite/Search.hs

```diff
@@ -41,8 +41,8 @@ import Kore.Internal.SideCondition (
 import Kore.Rewrite.RewritingVariable (
     RewritingVariableName,
  )
-import qualified Kore.Rewrite.SMT.Evaluator as SMT.Evaluator (
-    evaluate,
+import Kore.Rewrite.SMT.Evaluator (
```

Comment:
I'd prefer this to be qualified as SMT. Then it will be clear that the function actually sends the condition to be evaluated by the SMT.

Please do this in the other modules as well.

Reply:
Updated: https://github.com/kframework/kore/pull/2864/commits/a9503db5234221171d6ef7cfdb0d0cd7c51024b5